src/components/CustomTileModal/CustomTileModal.tsx

```diff
@@ -45,11 +45,24 @@ const CustomTileModal: React.FC<CustomTileModalProps> = ({
     const [displayHeader, setDisplayHeader] = useState(
         selectedItem?.displayHeader ?? '',
     )
+    const [errorMessage, setErrorMessage] = useState<boolean>(false)
 
     const [isSubmitAttempted, setIsSubmitAttempted] = useState(false)
 
+    // eslint-disable-next-line prettier/prettier
+    const urlPattern = new RegExp(
+        '^((http|https):\\/\\/)?www.([A-z0-9]+).([A-z]{2,})',
+    )
```

Comment:
Siden dette er et statisk pattern kan vi bruke /regex/ syntax: [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions), da slipper vi dobbelt escaped slash e.g. "\\\\/"

Også må "." escapes "\\." for å ikke bli en wildcard character